ripple/src/main/webapp/businessdetails.js

```diff
@@ -52,4 +52,172 @@ function removeColorFromRating(ratingType, rating, maxRating, elementIdPrefix, a
 /* TODO: Function will add new review to firestore, then refresh the page to see changes */
 function newReview() {
   location.reload();
+  clearLocalStorage(["starRating", "priceRating"]);
+}
+
+/* Load all business details on the body onload */
+function bdOnload() {
+  // TEMPORARY LOCAL STORAGE UNTIL MANAGE BUSINESS INFO PAGE IS COMPLETE
+  localStorage.setItem("walkingTime", 15); 
+  localStorage.setItem("walkingDist", 0.7);
+  localStorage.setItem("businessId", "uxq1XtM4zIHqxmdnjnf0");
+  localStorage.setItem("inputAddress", "398 S Willaman Dr, Los Angeles, CA 90048");
+  walkingDist = localStorage.getItem("walkingDist");
+  businessId = localStorage.getItem("businessId");
+  inputAddress = localStorage.getItem("inputAddress");
+  // Access 'businesses' collection to grab businessAddress + render maps + add businessName 
```

Comment:
what is this comment referring to?

Reply:
It's for logic that will be done. I accidentally forgot to add a "TODO" at the start of the comment